.dotnet/src/OpenAI.csproj

```diff
@@ -4,12 +4,13 @@
     <AssemblyTitle>SDK Code Generation OpenAI</AssemblyTitle>
     <Version>1.0.0-beta.1</Version>
     <PackageTags>OpenAI</PackageTags>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
We should keep targeting .NET 6.

Reply:
Oh, good catch! The `Update-Client` script keeps regenerating this file with this weird change, for some reason; I missed reverting it.